packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -285,6 +285,43 @@ export class JavaBuilder implements TargetBuilder {
 
     const profileName = 'local-jsii-modules';
     const localRepository = this.options.arguments['maven-local-repository'];
+
+    const repos = localRepos.map((repo) => ({
+      id: repo.replace(/[\\/:"<>|?*]/g, '$'),
+      url: `file://${repo}`,
+    }));
```

Comment:
We could simplify this a bit I feel...

```suggestion
    const repos = localRepos.map((repo, idx) => ({
      id: `_local_${idx}`,
      url: `file://${repo}`,
    }));
```

If doing this, we can ensure "no collisions" by documenting & encoding that "extra" repo IDs may not have a name starting with `_`.